bot/commands/data.js

```diff
@@ -3,72 +3,179 @@ import { currentSeason } from "../globals.js";
+            .setDescription("newest season to get replays from")
+        )
+    )
+    .addSubcommand((subcommand) =>
+      subcommand
+        .setName("team")
+        .setDescription("Get a team's player data")
+        .addRoleOption((option) =>
+          option
+            .setName("team")
+            .setDescription("the team to get data for")
+            .setRequired(true)
+        )
+    ),
+  async execute(interaction) {
+    switch (interaction.options.getSubcommand()) {
+      case "scout":
+        await scoutPlayer(interaction);
+        break;
+      case "team":
+        await getTeamData(interaction);
+        break;
     }
-}
+  },
+};
 
-async function scoutPlayer(interaction) {
-    async function dataCollector(interaction) {
-        const player = interaction.options.getUser('player');
-        const startSeason = interaction.options.getNumber('from_season') || 0;
-        const endSeason = interaction.options.getNumber('through_season') || currentSeason.number;
+async function getTeamData(interaction) {
+  async function dataCollector(interaction) {
+    const team = interaction.options.getRole("team");
 
-        const replaysByWeek = await loadReplays(startSeason, endSeason, player.id);
-        const replays = replaysByWeek.flatMap(week => [week.game1, week.game2, week.game3, week.game4, week.game5]).filter(game => !!game);
+    const playerData = await loadTeamData(team.id);
 
-        return { playerSnowflake: player.id, startSeason, endSeason, replays };
-    }
+    return { team, playerData };
+  }
 
-    function verifier(data) {
-        const { playerSnowflake, startSeason, endSeason, replays } = data;
-        let failures = [];
+  function verifier(data) {
+    const { team, playerData } = data;
+    let failures = [];
+
+    if (playerData.length === 0) {
+      failures.push(
+        `No data found for ${userMention(team.id)} from the current season.`
+      );
+    }
 
-        if (startSeason > endSeason) {
-            failures.push(`You asked for replays ${italic('after')} season ${startSeason} but ${italic('before')} season ${endSeason}. Obviously there are none.`);
-        }
+    return failures;
+  }
 
-        if (replays.length === 0) {
-            failures.push(`No replays found for ${userMention(playerSnowflake)} between seasons ${startSeason} and ${endSeason}.`)
-        }
+  function responseWriter(data) {
+    const { team, playerData } = data;
 
-        return failures;
+    function makePlayerIntoRow(p) {
+      return p.name
+        .padEnd(29, " ")
+        .concat("|     ")
+        .concat(p.wins)
+        .concat("|         ")
+        .concat(p.act_wins)
+        .concat("|       ")
+        .concat(p.losses)
+        .concat("|       ")
+        .concat(p.act_losses)
+        .concat("|         ")
+        .concat(p.ties)
+        .concat("|")
+        .concat(p.star_points.toString().padStart(13, " "))
+        .concat("|   ")
+        .concat(Math.ceil(100 * p.stars) / 100);
```

Comment:
```suggestion
      return `${rightAlign(29, p.name)}|${rightAlign(6, p.wins)}|${rightAlign(9, p.act_wins)}|${rightAlign(8, p.losses)}|${rightAlign(12, p.act_losses)}|${rightAlign(6, p.ties)}|${rightAlign(13, fixFloat(p.star_points)}| {fixFloat(p.stars)}
```

To match how I've written the table formatter in prettyTextStanding in season.js